buildmodel/commands.go

```diff
@@ -328,7 +328,7 @@ func SubmitUpdatePR(f *PRFlags) error {
 	runOrPanic(newGitCmd("commit", "-m", commitMessage))
 
 	// Push the commit.
-	args := []string{"push", *f.origin, b.PRBranchRefspec()}
+	args := []string{"push", auther.InsertAuth(*f.origin), b.PRBranchRefspec()}
```

Comment:
What if this is a dry run without auth?

Maybe assign `auther` to be a `NoAuther` in the error case to avoid nils?